reactions/language/src/main/ecore/TopLevelElements.ecore

```diff
@@ -61,6 +62,10 @@
         eType="#//ReactionsSegment" eOpposite="#//ReactionsSegment/routines"/>
     <eStructuralFeatures xsi:type="ecore:EReference" name="updateBlock" eType="#//UpdateBlock"
         containment="true"/>
+	<eStructuralFeatures xsi:type="ecore:EReference" name="logBlockBeforMatch" eType="#//LogBlock" containment="true" lowerBound="0" upperBound="1"/>
```

Comment:
Is there a reason to name it BeforMatch instead of BeforeMatch to be consistent with the others?